recipe/build.sh

```diff
@@ -3,7 +3,6 @@ cd build
 
 cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
       -DCMAKE_BUILD_TYPE=Release \
-      -DLLVM_TARGETS_TO_BUILD=host \
```

Comment:
Can you do the same in windows?

Reply:
It's already like this: https://github.com/conda-forge/llvmdev-feedstock/blob/2f5ed79c873ee4395dbc3d715868a542b8877022/recipe/bld.bat